### PR TITLE
Listings favourites

### PR DESCRIPTION
**Pull request by andr-dev**

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt [hunk 1]
```diff
@@ -120,16 +123,15 @@ class CreateListingPageViewModel @Inject constructor(
 							description = it.description,
 							residenceType = ResidenceType.house,
 							imgIds = imgIds,
-							roomsAvailable = it.numBedrooms,
-// TODO: ADDING LOCATION
-							latitude = 0f,
-							longitude = 0f,
-//							TODO: Update UI to allow the inputs of these newly added values
+							latitude = location!!.latitude.toFloat(),
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt]: Plz, let's not crash the app. Plz.

File: android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt [hunk 2]
```diff
@@ -120,16 +123,15 @@ class CreateListingPageViewModel @Inject constructor(
...
+							longitude = location.longitude.toFloat(),
+							// TODO: Update UI to allow the inputs of these newly added values
 							bathroomsAvailable = 0,
 							bathroomsEnsuite = 0,
 							bathroomsTotal = 0,
+							roomsAvailable = 0,
 							roomsTotal = 0,
-							gender = "Female",
+							gender = Resources.getSystem().getString(Gender.OTHER.stringId),
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt:132]: I assume that we'll fill this later so I think the string literal as a placeholder is fine.

**Review by leeminkyu16 — APPROVED** [review]
Just a few small things.
Let's Gamble, Try Merging

File: android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt (hunk 1 above)
Comment (Jiay0928) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt]: I think something went wrong during rebasing? The latitude/longtitude = 0 is the correct value on master.

**Pull request merged**